From ace27a90c52d75dd09ce41d282c305dbb673fea3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Wed, 11 Jan 2017 10:14:31 +0000
Subject: [PATCH] Resolves: rhbz#1408500 gtk3 toolbar borders too distinct vs
 native gtk3 ones

Change-Id: I7ce284d3e6f999e2ac08326dc40b060bf70d7d4b
(cherry picked from commit 5088b2066651ed133dcfd93388f7ac7187844d33)
---
 vcl/unx/gtk3/gtk3salnativewidgets-gtk.cxx | 1 +
 1 file changed, 1 insertion(+)

diff --git a/vcl/unx/gtk3/gtk3salnativewidgets-gtk.cxx b/vcl/unx/gtk3/gtk3salnativewidgets-gtk.cxx
index cdf4f08..8deb58b 100644
--- a/vcl/unx/gtk3/gtk3salnativewidgets-gtk.cxx
+++ b/vcl/unx/gtk3/gtk3salnativewidgets-gtk.cxx
@@ -2908,6 +2908,7 @@ void GtkData::initNWF()
     ImplSVData* pSVData = ImplGetSVData();
     pSVData->maNWFData.mbFlatMenu = true;
     pSVData->maNWFData.mbCheckBoxNeedsErase = true;
+    pSVData->maNWFData.mbDockingAreaAvoidTBFrames = true;
     pSVData->maNWFData.mbCanDrawWidgetAnySize = true;
     pSVData->maNWFData.mbDDListBoxNoTextArea = true;
     pSVData->maNWFData.mbNoFocusRects = true;
-- 
2.9.3

